src/popup/components/Protocol/EnsureUserSavedMnemonic.tsx

```diff
@@ -4,10 +4,13 @@ import {
   Title,
   Subtitle,
   ClickableText,
+  Cta,
 } from "@popup/components/Protocol/common";
-import React, { useEffect } from "react";
+import { useEffect } from "react";
 import styled from "styled-components";
 import { getProtocolOptIn } from "@services/Factory";
+import { SetupSuccess } from "./SetupScreen";
```

Comment:
The name `SetupSuccess` is confusing in this context. Maybe rename it with import as? `import { SetupSuccess as ShowMnemonic }`?